iroh-relay/src/protos/relay.rs

```diff
@@ -41,8 +41,17 @@ const MAX_FRAME_SIZE: usize = 1024 * 1024;
 /// The Relay magic number, sent in the FrameType::ClientInfo frame upon initial connection.
 const MAGIC: &str = "RELAY🔑";
 
+/// Maximum time for a relay server to respond to a relay protocol ping.
 #[cfg(feature = "server")]
-pub(crate) const KEEP_ALIVE: Duration = Duration::from_secs(60);
+pub(crate) const PING_TIMEOUT: Duration = Duration::from_secs(5);
+
+/// Interval in which we ping the relay server to ensure the connection is alive.
```

Comment:
This is used by the client to ping the server.  But maybe just make it pub so the magicsock can import this too and it's only defined once?

Reply:
fixed